src/installers/minishift-addon.ts

```diff
@@ -61,6 +71,18 @@ export class MinishiftAddonHelper {
     ], {renderer: flags['listr-renderer'] as any})
   }
 
+  async grabVersion(): Promise<number> {
+    let args = ['version']
+    const { stdout} = await execa('minishift',
+                                     args,
+                                     {reject: false })
+    if (stdout) {
+      return parseInt(stdout.replace(/\D/g, '').substring(0, 3), 10)
```

Comment:
Can you please add a test for `grabVersion`. That would really help if we need to refactor or update the parsing mechanism.

Reply:
I've added tests

Reply:
@benoitf thanks